back/app/controllers/web_api/v1/ideas_controller.rb

```diff
@@ -150,6 +150,10 @@ def create
     service.before_create(input, current_user)
 
     authorize input
+    if anonymous_not_allowed?(participation_context)
```

Comment:
Think for we should be calling anonymous_not_allowed earlier in the method - applies to the other methods and controllers too

Reply:
That is what I did initially. However this results in a Pundit Authorization Not Performed error. It's too bad this error is still raised even if the error status is already unauthorized.

Reply:
Hmm OK, no problem leave it as it is then